projects/igniteui-angular/migrations/update-11_1_0/changes/members.json

```diff
@@ -363,6 +363,48 @@
             "definedIn": [
                 "IgxTooltipTargetDirective"
             ]
+        },
```

Comment:
If that pr is targeting master (12.0.x)v, we have to move those migration changes.